src/bin/nydusd/fs_cache.rs

```diff
@@ -538,11 +573,26 @@ impl FsCacheHandler {
                     }
                     Ok(md) => {
                         let cache_file = unsafe { File::from_raw_fd(msg.fd as RawFd) };
+                        let bootstrap_fd = f.as_raw_fd();
+                        let cachefile_fd = cache_file.as_raw_fd();
                         let object = FsCacheObject::Bootstrap(Arc::new(FsCacheBootStrap {
                             bootstrap_file: f,
                             cache_file,
                         }));
                         e.insert((object, msg.fd));
+                        let len = md.len() as usize;
+                        ASYNC_RUNTIME.spawn_blocking(move || {
+                            //add slight delay to let copen reply first
+                            thread::sleep(time::Duration::from_millis(10));
+                            for _i in 0..3 {
+                                if let Ok(()) =
```

Comment:
Why try 4 times rather than 8 ?

Reply:
why we need retry 8 times ?

Reply:
I mean is there theory based to define it as 4? Or it is picked randomly?

Reply:
just picked randomly :)